app/controllers/declaration/OfficeOfExitController.scala

```diff
@@ -106,9 +106,7 @@ class OfficeOfExitController @Inject()(
     declarationType match {
       case DeclarationType.SUPPLEMENTARY | DeclarationType.STANDARD =>
         controllers.declaration.routes.TotalNumberOfItemsController.displayPage
-      case DeclarationType.CLEARANCE =>
-        controllers.declaration.routes.TotalPackageQuantityController.displayPage
-      case DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL =>
+      case DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL | DeclarationType.CLEARANCE =>
```

Comment:
Is this correct place? Based on the navigator change there is a `NatureOfTheTransactionController`. Shouldn't this change be in place where previously we redirected to this controller?

Reply:
It's discussed. This is correct